src/Directory.Packages.props

```diff
@@ -1,14 +1,13 @@
 <Project>
-
```

Comment:
Please don't remove readability spacing in project(-like) files.

Reply:
Good eye! Pushed a fix. This was likely the IDE doing this.